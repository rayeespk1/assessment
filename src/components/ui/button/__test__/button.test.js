import React from "react";
import '@testing-library/jest-dom'
import { render, cleanup } from "@testing-library/react";
import Button from "../button";

afterEach(cleanup);

test("renders", () => {
    const { asFragment } = render(<Button btnType="Success" />);
    expect(asFragment()).toMatchSnapshot();
});

test("showing backdrop", () => {
    const { getByTestId } = render(<Button btnType="Success" />);
    expect(getByTestId("buttonTag")).toHaveClass("Button Success");
});