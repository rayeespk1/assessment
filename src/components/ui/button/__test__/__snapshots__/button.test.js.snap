// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <button
    class="Button Success"
    data-testid="buttonTag"
  />
</DocumentFragment>
`;
